tests/Admin/AdminHelperTest.php

```diff
@@ -90,29 +95,11 @@ public function testAddNewInstance(): void
         $this->helper->addNewInstance($object, $fieldDescription);
     }
 
-    public function testAddNewInstanceWithParentAssociation(): void
-    {
-        $admin = $this->createMock(AdminInterface::class);
-        $admin->expects($this->once())->method('getNewInstance')->willReturn(new \stdClass());
-
-        $fieldDescription = $this->createMock(FieldDescriptionInterface::class);
-        $fieldDescription->expects($this->once())->method('getAssociationAdmin')->willReturn($admin);
-        $fieldDescription->expects($this->once())->method('getAssociationMapping')->willReturn(['fieldName' => 'fooBar']);
-        $fieldDescription->expects($this->once())->method('getParentAssociationMappings')->willReturn([['fieldName' => 'parent']]);
-
-        $object2 = $this->getMockBuilder(\stdClass::class)
-            ->setMethods(['addFooBar'])
-            ->getMock();
-        $object2->expects($this->once())->method('addFooBar');
-
-        $object1 = $this->getMockBuilder(\stdClass::class)
-            ->setMethods(['getParent'])
-            ->getMock();
-        $object1->expects($this->once())->method('getParent')->willReturn($object2);
-
-        $this->helper->addNewInstance($object1, $fieldDescription);
-    }
-
+    /**
+     * @group legacy
```

Comment:
A `NEXT_MAJOR` comment could help to understand if this method or if the "legacy" group must be removed.

Reply:
Sure, done.